picard/ui/mainwindow.py

```diff
@@ -407,10 +407,10 @@ def create_actions(self):
         self.add_files_action.setShortcut(QtGui.QKeySequence.Open)
         self.add_files_action.triggered.connect(self.add_files)
 
-        self.add_directory_action = QtWidgets.QAction(icontheme.lookup('folder'), _("A&dd Folder..."), self)
+        self.add_directory_action = QtWidgets.QAction(icontheme.lookup('folder'), _("Add Fold&er..."), self)
```

Comment:
Do we really need to change the accelerator here? This will be different for different languages anyway, and it does not necessarily correspond to the shortcut.

Reply:
I just wanted to use common letter `e` to match Ctrl+E, in many languages it is possible to keep it. It isn't mandatory, and of course it will depend on translations, but since we are at it, I thought it would be more consistent.

Reply:
Ok, makes sense. The annoying part here is that it breaks translations for no good reason :) But that's something we can handle. I'll go through existing translations and approve this string manually.